lm-agent-snap/README.md

```diff
@@ -0,0 +1,76 @@
+# lm-agent-snap
+The License Managar Agent bundled into a Snap.
+
+# Installation instructions
+
+For installing from the remote store, run:
```

Comment:
What do you think of having this say "Snap store" instead and make it a link?

Reply:
done in [13c89e3](https://github.com/omnivector-solutions/license-manager/pull/389/commits/13c89e37e4fca4d4f8cd3de0090c4f227cc7a738)